Src/DependencyCollector/NetCore/DependencyCollector.NetCore.csproj

```diff
@@ -17,6 +18,9 @@
     <GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
     <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard1.3' ">
+    <DefineConstants>$(DefineConstants);NETSTANDARD1_3</DefineConstants>    
```

Comment:
I don't believe this is necessary because you get it 'for free' from the default targets.